---
title: "A comprehensive proteomic and bioinformatics analysis of human spinal cord injury plasma identifies proteins associated with the complement cascade and liver function as potential prognostic indicators of neurological outcome"
# output:
#   rticles::arxiv_article:
#     pandoc_args:
#       - --filter=pandoc-crossref
# authors:
#   - name: Gabriel Mateus Bernardo Harrington
#     affiliation: Keele Univeristy
#     email: g.m.bernardo.harrington@keele.ac.uk
#   - name: Paul Cool
#     affiliation: Robert Jones and Agnes Hunt Orthopaedic Hospital
#     email: paul.cool@nhs.net
#   - name: Charlotte Hulme
#     affiliation: Keele Univeristy
#     email: charlotte.hulme@nhs.net
#   - name: Jessica Fisher-Stokes
#     affiliation: Keele Univeristy
#     email: j.b.fisher-stokes@keele.ac.uk
#   - name: Mandy Peffers
#     affiliation: University of Liverpool
#     email: m.j.peffers@liverpool.ac.uk
#   - name: Wagih El Masri
#     affiliation: Robert Jones and Agnes Hunt Orthopaedic Hospital
#     email: bellstonehse@btinternet.com
#   - name: Aheed Osman
#     affiliation: Robert Jones and Agnes Hunt Orthopaedic Hospital
#     email: aheed.osman@nhs.net
#   - name: Joy Roy Chowdhury
#     affiliation: Robert Jones and Agnes Hunt Orthopaedic Hospital
#     email: joy.chowdhury@nhs.net
#   - name: Naveen Kumar
#     affiliation: Robert Jones and Agnes Hunt Orthopaedic Hospital
#     email: n.kumar1@nhs.net
#   - name: Srinivasa Budithi
#     affiliation: Robert Jones and Agnes Hunt Orthopaedic Hospital
#     email: sbudithi@nhs.net
#   - name: Karina Wright
#     affiliation: Keele Univeristy
#     email: karina.wright1@nhs.net
output:
  bookdown::pdf_document2:
    toc: no
    #number_sections: yes
    keep_tex: yes
    df_print: kable
    # template: extras/elife-template.tex
    # pandoc_args:
    #   - --filter=pandoc-crossref
    #   - --lua-filter=extras/scholarly-metadata.lua
    #   - --lua-filter=extras/author-info-blocks.lua
    #   - --template=extras/mytemplate.tex
  bookdown::word_document2:
    toc: no
    number_sections: yes
    df_print: kable
#documentclass: elife
#classoption: 9pt,lineno
# author:
#  - name: Gabriel Mateus Bernardo Harrington, PhD
#    email: g.m.bernardo.harrington@keele.ac.uk
#    #telephone: 0169140-4138
#    #address: Some Institute of Technology
#    institute: [UofK, CU]
#  - name: Paul Cool, MD
#    email: paul.cool@nhs.net
#    institute: [RJAH, UofK]
#  - name: Charlotte Hulme, PhD
#    email: charlotte.hulme@nhs.net
#    institute: [UofK]
#  - name: Jessica Fisher-Stokes, MRes
#    email: j.b.fisher-stokes@keele.ac.uk
#    institute: [UofK]
#  - name: Mandy Peffers, PhD
#    email: m.j.peffers@liverpool.ac.uk
#    institute: [UoL]
#  - name: Wagih El Masri, MD
#    email: bellstonehse@btinternet.com
#    institute: [RJAH]
#  - name: Aheed Osman, MBChB
#    email: aheed.osman@nhs.net
#    institute: [RJAH]
#  - name: Joy Roy Chowdhury, MBBS
#    email: joy.chowdhury@nhs.net
#    institute: [RJAH]
#  - name: Naveen Kumar, MBBS
#    email: n.kumar1@nhs.net
#    institute: [RJAH]
#  - name: Srinivasa Budithi, MD
#    email: sbudithi@nhs.net
#    institute: [RJAH]
#  - name: Karina Wright, PhD
#    email: karina.wright1@nhs.net
#    institute: [UofK, RJAH]
#    corresponding_author: yes   
# institute:
#   - UofK: Keele University, Staffordshire, United Kingdom, ST5 5BG
#   - RJAH: Robert Jones and Agnes Hunt Orthopaedic Hospital NHS Foundation Trust, Oswestry, United Kingdom, SY10 7AG
#   - UoL: University of Liverpool, Liverpool, United Kingdom, L69 7ZX
#   - UC: Cardiff University, Cardiff, Wales, United Kingdom, CF10 3AT
author:
 - Gabriel Mateus Bernardo Harrington, PhD^[Keele University]
 - Paul Cool, MD^[Robert Jones and Agnes Hunt Orthopaedic Hospital NHS Foundation Trust; Keele University]
 - Charlotte Hulme, PhD
 - Jessica Fisher-Stokes, MRes
 - Mandy Peffers, PhD
 - Wagih El Masri, MD
 - Aheed Osman, MBChB
 - Joy Roy Chowdhury, MBBS
 - Naveen Kumar, MBBS
 - Srinivasa Budithi, MD
 - Karina Wright, PhD^[karina.wright1@nhs.net]
keywords:
  - Spinal cord injury
  - Biomarker
  - Proteomics
  - Complement cascade
#link-citations: yes
#csl: extras/elife.csl
# geometry: margin=2cm
always_allow_html: true
date: "`r format(Sys.Date())`"
bibliography: extras/paper_and_pkgs.bib
#biblio-style: unsrt
header-includes:
   - \usepackage{longtable}
   - \usepackage{array}
   - \usepackage{pdflscape}
   - \newcommand{\blandscape}{\begin{landscape}}
   - \newcommand{\elandscape}{\end{landscape}}
   - \usepackage{float} \floatplacement{figure}{H} 
   - \newcommand{\beginsupplement}{\setcounter{table}{0}  \renewcommand{\thetable}{S\arabic{table}} \setcounter{figure}{0} \renewcommand{\thefigure}{S\arabic{figure}}}
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE) #sets these options for whole document
options(knitr.kable.NA = '') #sets NAs in tables to be blank
options(digits = 4) #sets rounding for the document

# below fixes kable tables in word output
# options(kableExtra.auto_format = FALSE)
## keep aux file
## options(tinytex.clean = FALSE)
```

```{r load-packages, include=FALSE}
## create vector of packages used
pkg <- c("kableExtra", "tidyverse", "captioner", "bibtex",
         "Hmisc", "psych", "naniar", "data.table", "rmarkdown",
         "knitr", "BiocManager", "purrr", "base", "readxl", "cowplot",
         "patchwork", "lubridate", "DiagrammeR", "ggVennDiagram",
         "RColorBrewer", "lime", "bookdown", "stringr", "rlang")

pkg_bioconductor <- c("ReactomePA", "STRINGdb", "MSstats")

## create function to installing any missing packages and then load them all
sort_packages <- function(packages, bioconductor = FALSE) {
    ## Check if packages are not installed and assign the
    ## names of the packages not installed to the variable new.pkg
    new_pkg <- packages[!(packages %in% installed.packages())]
    ## If the packages are bioconductor, install via biocmanager
    ## If there are any packages in the list that aren't installed,
    ## install them
    if (length(new_pkg) & !bioconductor) {
        install.packages(new_pkg, repos = "https://cran.rstudio.com")
    } else if (length(new_pkg) & bioconductor) {
        BiocManager::install(new_pkg)
    }
    ## load packages
    lapply(packages, require, character.only = TRUE)
}
## use function
sort_packages(pkg)
sort_packages(pkg_bioconductor, bioconductor = TRUE)
```

```{r, label='add-packages-to-bib'}
## get the packages version
packages_versions <- function(p) {
    paste(packageDescription(p)$Package, packageDescription(p)$Version, sep = " ")
}
## Get the packages references
write.bib(c(pkg, pkg_bioconductor), "extras/packages.bib")
## merge the zotero references and the packages references
cat(paste("% Automatically generated", Sys.time()), "\n% DO NOT EDIT",
{ readLines("extras/paper.bib") %>%
      paste(collapse = "\n") },
{ readLines("extras/packages.bib") %>%
      paste(collapse = "\n") },
file = "extras/paper_and_pkgs.bib",
    sep = "\n")
## Some packages reference keys must be modified
## (their key is not the package name)
## check in packages.bib
packages_keys <- c(pkg, pkg_bioconductor) %>%
  enframe() %>%
  mutate(value = case_when(value == "knitr" ~ "@knitr1",
                           #value == "boot" ~ "@boot1",
                           TRUE ~ paste0("@", value)))
```

```{r ggplot-theme3}
custom_theme <- function() {
theme_bw() +
theme(
axis.text = element_text(size = 8),
## axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1),
axis.title = element_text(size = 8),
panel.grid.major = element_blank(),
panel.grid.minor = element_blank(),
panel.background = element_rect(fill = '#ffffff'),
strip.background = element_rect(fill = 'gray', color = 'black', size = 0.5),
strip.text = element_text(face = 'bold', size = 8, color = 'white'),
legend.position = 'right',
legend.justification = 'center',
legend.background = element_blank(),
panel.border = element_rect(color = 'grey5', fill = NA, size = 0.5))
#scale_fill_manual(values = c("tomato", "steelblue"))) # if there's a issue with the graphs remove this line
}

theme_set(custom_theme())
```

# Abstract

## Introduction

Spinal Cord Injury (SCI) is a major cause of disability, with complications post-injury often leading to life-long health issues with need of extensive treatment.
Neurological outcome post-SCI can be variable and difficult to predict, particularly in incomplete injured patients.
The identification of specific SCI biomarkers in blood, may be able to improve prognostics in the field.
This study has utilised proteomic and bioinformatics methodologies to investigate differentially expressed proteins in plasma samples across human SCI cohorts with the aim of identifying prognostic biomarkers and biological pathway alterations that relate to neurological outcome.

## Methods and Materials

Blood samples were taken, following informed consent, from ASIA impairment scale (AIS) grade C "Improvers" (those who experienced an AIS grade improvement) and "Non-Improvers" (No AIS change), and AIS grade A and D at \<2 weeks ("Acute") and approx. 3 months ("Sub-acute") post-injury.
The total protein concentration from each sample was extracted, with pooled samples being labelled and non-pooled samples treated with ProteoMiner™ beads.
Samples were then analysed using two 4-plex isobaric tag for relative and absolute quantification (iTRAQ) analyses and a label-free experiment for comparison, before quantifying with mass spectrometry.
Data are available via ProteomeXchange with identifiers PXD035025 and PXD035072 for the iTRAQ and label-free experiments respectively.

Proteomic datasets were analysed using OpenMS (version 2.6.0).
R (version 4.1.4) and in particular, the R packages MSstats (version 4.0.1) and pathview (version 1.32.0) were used for downstream analysis.
Proteins of interest identified from this analysis were further validated by enzyme-linked immunosorbent assay (ELISA).

## Results

The data demonstrated proteomic differences between the cohorts, with the results from the iTRAQ approach supporting those of the label-free analysis.
A total of 79 and 87 differentially abundant proteins across AIS and longitudinal groups were identified from the iTRAQ and label-free analyses, respectively.
Alpha-2-macroglobulin (A2M), retinol binding protein 4 (RBP4), serum amyloid A1 (SAA1), Peroxiredoxin 2, alipoprotein A1 (ApoA1) and several immunoglobulins were identified as biologically relevant and differentially abundant, with potential as individual prognostic biomarkers of neurological outcome.
Bioinformatics analyses revealed that the majority of differentially abundant proteins were components of the complement cascade and most interacted directly with the liver.

## Conclusions

Many of the proteins of interest identified using proteomics were detected only in a single group and therefore have potential as a binary (present or absent) biomarkers, RBP4 and PRX-2 in particular.
Additional investigations into the chronology of these proteins, and their levels in other tissues (cerebrospinal fluid in particular) are needed to better understand the underlying pathophysiology, including any potentially modifiable targets.
Pathway analysis highlighted the complement cascasde as being significant across groups of differential functional recovery.

# Introduction

Spinal cord injury (SCI) is the transient or permanent loss of normal spinal sensory, motor or autonomic function, and is a major cause of disability.
Globally, SCI affects around 500,000 people each year and is most commonly the result of road traffic accidents or falls.[@crozier-shaw_management_2020] Patients typically require extensive medical, rehabilitative and social care at high financial cost to healthcare providers. The lifetime cost of care in the UK is estimated to be £1.12 million (mean value) per SCI, with the total cost of SCI in the UK to the NHS being £1.43 billion in 2016.[@mcdaid_understanding_2019] Individuals with SCI show markedly higher rates of mental illness relative to the general population.[@furlan_health_2017] Complications arising post-SCI can be long-lasting and often include pain, spasticity and cardiovascular disease, where the systemic inflammatory response that follows SCI can frequently result in organ complications, particularly in the liver and kidneys.[@gris_systemic_2008; @sun_multiple_2016; @hagen_acute_2015]

The recovery of neurological function post-SCI is highly variable, requiring any clinical trials to have an impractically large sample size to prove efficacy, hence the translation of novel efficacious therapies is challenging and expensive.[@spiess_conversion_2009] Being able to more accurately predict patient outcomes would aid clinical decisions and facilitate future clinical trials.
Therefore, novel biomarkers that allow for stratification of injury severity and capacity for neurological recovery would be of high value to the field.

Biomarkers studies in SCI often investigate protein changes in cerebral spinal fluid (CSF) as the closer proximity of this medium is thought to be more reflective of the parenchymal injury.[@kwon_neurochemical_2019; @hulme_developing_2017] Whilst this makes CSF potentially more informative for elucidating the pathology of SCI, the repeated use of CSF for routine analysis presents challenges in clinical care due to the risk and expense associated with the invasiveness of the collection procedure.
In contrast, systemic biomarkers measurable in the blood represent a source of information that can be accessed and interpreted both a lower cost and risk.
Studies of traumatic brain injury have demonstrated that protein markers identified in CSF are also detectable in both plasma and serum.[@wang_update_2018] More recently, circulating white blood cell populations have also been identified as potential SCI injury biomarkers, with a 2021 study showing that elevated levels of neutrophils were associated with no AIS grade conversion, while conversely an increase in lymphocytes during the first week post-SCI were associated with an AIS grade improvement.[@jogia_peripheral_2021]

A number of individual proteins have been shown to be altered in the bloods post-SCI, including multiple interleukins (IL), tumour necrosis factor alpha (TNF-$\alpha$) and C-reactive protein (CRP).[@segal_circulating_1997; @hayes_elevated_2002; @frost_inflammatory_2005]

Further, changes in inflammatory marker levels detected in acute SCI patients were found to be mirrored in donor-matched blood and CSF, albeit at lower absolute concentrations systemically.[@kwon_cerebrospinal_2010]

Previously, we have shown that routinely collected blood measures associated with liver function and inflammation added predictive value to AIS motor and sensor outcomes at discharge and 12-months post-injury.[@bernardo_harrington_routinely_2020; @brown_preliminary_2019] The current study uses an unbiased shotgun proteomic approach to investigate differentially expressed proteins in SCI patients, coupled with bioinformatics pathway and network analyses.

# Methods and Materials

```{r, label='load-patient-demo-data-chap3'}
## read in data
proteom_demo <- read_excel("data/proteomics-patients_2021-03-26.xlsx", sheet = 2) %>%
    janitor::clean_names()
## get age at injury and general injury level
proteom_demo$age_at_injury <- time_length(difftime(proteom_demo$doi, proteom_demo$dob), "years")
proteom_demo$injury_level <- substr(proteom_demo$injury_level, 1, 1)
## ais change col
proteom_demo$ais_change <- ifelse(proteom_demo$ais_grade_initial == proteom_demo$ais_grade_latest,
                                  proteom_demo$ais_grade_initial,
                                  paste(proteom_demo$ais_grade_initial, "->",
                                        proteom_demo$ais_grade_latest, sep = ""))
## remove uneeded cols
prot_demo <- proteom_demo %>%
    dplyr::select(!c(hos_num, dob, doi))
```

## Patients

Blood samples were taken from SCI patients who had provided informed consent and in accordance to ethical provided by the National Research Ethics Service [NRES] Committee North West Liverpool East [11/NW/0876].
"Improvers" were defined as individuals who experienced an AIS grade improvement from admission to a year post-injury, whereas "non-improvers" were defined as patients who saw no change in AIS grade in the same period (Table \@ref(tab:patient-demo-chap3)).

```{r, label='patient-demo-chap3'}
## summarise params function
demo_summary <- function(df, group_var) {
  df2 <- df %>%
    dplyr::group_by(.data[[group_var]]) %>%
    dplyr::summarise(n = n(), group = group_var) %>%
    dplyr::rename(summary = all_of(group_var)) %>%
    dplyr::mutate(percent = round(n / sum(n) * 100))
  return(df2)
}
## use function
demo_sum <- map(names(prot_demo)[-c(2, 6:8)], ~ demo_summary(prot_demo, .x))
## get median age
age_sum <- data.frame(summary = "Age at injury \n(Median years\u00B1IQR)",
                      n = paste(round(median(prot_demo$age_at_injury)),
                                "\u00B1",
                                round(IQR(prot_demo$age_at_injury)), sep = ""),
                      group = "age_at_injury",
                      percent = "-")
## rbind
demo_sum <- do.call(rbind, demo_sum)
demo_sum <- rbind(demo_sum, age_sum)
## replace NA with "no"
demo_sum$summary[is.na(demo_sum$summary)] <- "No"
demo_sum$summary <- ifelse(demo_sum$summary == "1", "Yes",
                           demo_sum$summary)
## remove group
demo_sum <- demo_sum %>%
  dplyr::select(!group)

knitr::kable(demo_sum, "latex", longtable = TRUE, booktabs = TRUE,
             caption = "Patient demographics. \u00B1 denotes interquartile range",
             col.names = c("", "n", "Percent")) %>%
  ## column_spec(1, width = "4cm") %>%
  kable_styling(latex_options = c("repeat_header")) %>%
  pack_rows("Polytrauma", 1, 2) %>%
  pack_rows("Gender", 3, 4) %>%
  pack_rows("Diabetes", 5, 6) %>%
  pack_rows("Neurological level", 7, 9) %>%
  pack_rows("AIS change", 10, 13)
```

## Plasma collection and storage

Plasma samples were collected within 2 weeks of injury (acute) and at approximately 3 months post-injury (subacute).
Upon collection in EDTA (ethylenediaminetetraacetic acid) coated tubes samples were centrifuged at 600g for 15 minutes, to pellet erythrocytes and the resultant plasma fraction was aspirated and divided into aliquots for long-term storage in -80°C briefly and liquid nitrogen in the longer term.

## Sample preparation and analysis using iTRAQ proteomics {#itraq-sample-prep}

Thawed plasma samples ($2\mu l$) each were diluted with distilled water ($98\mu l$).
Total protein was quantified using a Pierce™ $660 nm$ Protein Assay (Thermo Fisher Scientific, Hemel Hempstead, UK)[@stoscheck_protein_1987].

A total of $100 mg$ of plasma protein was taken from each sample and pooled equally to form a patient test group.
For example, the AIS C improver group was pooled from 10 separate patient samples, 10mg of protein per patient.

The pooled plasma samples were precipitated by incubation of the sample in six times the volume of chilled acetone for 1 hour at -20°C.
The samples were then centrifuged at 6,000G for 10 minutes at 4°C, and re-suspended in $200\mu l$ of triethylammonium bicarbonate buffer.
Sequencing Grade Modified Trypsin ($10\mu g/85\mu g$ of protein; Promega, Madison, WI, USA) was then added to the samples for overnight digestion at 37°C.
Peptides underwent reduction and alkylation (according to the manufacturer's instructions; Applied Biosystems, Bleiswijk, The Netherlands).
Tryptic digests were labelled with iTRAQ tags (again according to the manufacturer's instructions for the iTRAQ kit), before being pooled into test groups and dried in a vacuum centrifuge.
Two individual iTRAQ experiments were set up, the first to assess acute and sub-acute improvers or non-improvers and the second to assess acute improvers and non-improvers to AIS grade A and D patients.
The following tags were used for each group of patient samples 114 tag - acute improvers, 115 tag - sub-acute improvers, 116 tag - acute non-improvers and 117 tag - sub-acute non-improvers for run 1 and 114 tag - acute improvers, 115 tag - acute non-improvers, 116 tag - AIS grade A and 117 tag - AIS grade D for run 2.

### iTRAQ mass spectrometry analysis

The samples were analysed at the BSRC St. Andrews University Mass Spectrometry and Proteomics Facility.
A total of 12 SCX fractions were analysed by nano-electrospray ionisation-liquid chromatography/tandem mass spectrometry (LC-MS/MS) using a TripleTOF 5600 tandem mass spectrometer (AB Sciex, Framingham, MA, USA) as described previously.[@fuller_stathmin_2015] 
Each fraction ($10 \mu l$) was then analysed by nanoflow LC-ESI-MSMS, as described previously.

The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD035025 and 10.6019/PXD035025.[@perez-riverol_pride_2021]

<!-- which reference would this be? Also does anything else from this method need adding in as descriptive? -->

```{=html}
<!-- Each fraction ($10 \mu l$) was then analysed by nanoflow LC-ESI–MSMS. The peptides were separated using a nanoLC Ultra 2D plus loading pump and nanoLC AS-2 autosampler chromatography system (Eksigent, Redwood City, CA, USA), using a PepMap RSLC column ($75\mu l$ x $15 cm$) and an Acclaim PepMap100 trap ($100\mu m$ x $2 cm$) (ThermoFisher Scientific, Waltham, MA, USA).
After washing the peptides on the trap column for 20 minutes at $5\mu L$ min$^{-1}$, the trap was switched in line with the column and the peptides eluted with a gradient of increasing MeCN from 95% buffer A (98% H~2~O, 2% MeCN, 0.1% FA), 5% buffer B (2% H~2~O, 98% MeCN, 0.1% FA) to 65% buffer A, 35% buffer B over 60 minutes, then to 50% buffer A, 50% buffer B over a further 20 minutes, before increasing the concentration of buffer B to 95% over a further 10 minutes.
The column was then washed with 95% buffer B before re-equilibration in 95% buffer A. A flow rate of $300 nL$ min$^{-1}$ was employed. The eluent was sprayed into a TripleTOF 5600 tandem mass spectrometer (ABSciex, Foster City, CA, USA), using a NANOSpray III source, and analyzed in Information Dependent Acquisition (IDA) mode, performing $250 ms$ of MS followed by $100 ms$ MSMS analyses on the 20 most intense peaks with a charge state of +2 to +5. 

Parent (MS) ions were accepted with a mass tolerance of 50 mDa and MSMS was conducted with a rolling collision energy (CE) inclusive of preset iTRAQ CE adjustments.
Analyzed parent ions were then excluded from analysis for 13 s after 3 occurrences.
-->
```

## Sample preparation and analysis using label-free proteomics {#label-free-sample-prep}

No sample pooling was used, and so each of the 73 samples were maintained separately throughout protein equalisation, mass spectrometry, and label-free quantification steps.
Thus, protein abundance was quantified for each sample, whereupon mean protein abundance across experimental groups was calculated to assess protein changes.

To reduce the dynamic range of proteins, ProteoMiner™ beads (BioRad, Hemel Hempstead, UK) were used.[@boschetti_proteominer_2008] Total protein was quantitated with a Pierce™ $660 nm$ Protein Assay (Thermo Fisher Scientific, Hemel Hempstead, UK), whereupon 5 mg of total protein was applied to ProteoMiner™ beads, and processed as described previously.[@stoscheck_protein_1987; @peffers_comprehensive_2015]

### Label free mass spectrometry analysis

Tryptic peptides were subjected to LC-MC/MC via a 2-h gradient on a NanoAcquity™ ultraperformance LC (Waters, Manchester, UK) connected to a Q-Exactive Quadrupole-Orbitrap instrument (Thermo-Fisher Scientific Hemel Hempstead, UK).

The Q-Exactive was operated in a data dependent positive electrospray ionisation mode, automatically switching between full scan MS and MS/MS acquisition.
Survey full scan MS spectra (*m/z* 300--2000) were acquired in the Orbitrap with 70,000 resolution (*m/z* 200) following accumulation of ions to $1\times 10^6$ target value based on the predictive automatic gain control values from the previous full scan.
Dynamic exclusion was set to 20s, the 10 most intense multiply charged ions ($z \geq 2$) were sequentially isolated and fragmented in the octopole collision cell by higher energy collisional dissociation (HCD), with a fixed injection time of 100ms and 35,000 resolution.
The following mass spectrometric conditions were used: spray voltage, 1.9kV, no sheath or axillary gas flow; normalised HCD collision energy 30%; heated capillary temperature, 250°C.
MS/MS ion selection threshold was set to $1\times 10^4$ count and 2Da isolation width was set.

The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD035072 and 10.6019/PXD035072.[@perez-riverol_pride_2021]

## iTRAQ OpenMS analysis {#openms-chap3}

TripleTOF 5600 tandem mass spectrometer output files produced in the ABSciex proprietary `.wiff` file format were converted to an open file format, `.mzML` for analysis with OpenMS (version 2.6.0).
The docker image of ProteoWizard version 3.0.20287 was used for conversion, and peak picking was applied on conversion [@chambers_cross-platform_2012].
OpenMS version 2.6.0 was used for further analysis.[@rost_openms_2016] Unless otherwise stated, default arguments were used.
The 12 fraction files were merged and sorted by retention time.
A decoy database was generated with `DecoyDatabase` and the `-enzyme` flag set to `Trypsin`, the human reference proteome was taken from Uniprot (Proteome ID: UP000005640, downloaded: 2020-10-01), as was the `.fasta` for porcine trypsin (Entry: P00761, downloaded: 2020-10-01).[@the_uniprot_consortium_uniprot_2021]

The `MSFGPlusAdapter` was used to run the search.
For the `-fixed_modifications` "Methylthio (C)" and "iTRAQ4plex (N-term)" were passed due to the alkylating agent used in sample preperation and to account for the N-terminus modifications made by iTRAQ tags.
"Oxidation (M)" was passed to `-variable_modifications` to reflect the likely occurrence of methionine oxidation.
To reflect the instrument the following flags were also set: `-precursor_mass_tolerance 20 -enzyme Trypsin/P -protocol iTRAQ -instrument high_res`.

To annotate the search results `PeptideIndexer` and `PSMFeatureExtractor` were used.
For peptide level score estimation and filtering `PercolatorAdapter` was used with the following arguments: `-score_type q-value -enzyme trypsinp`.
`IDFilter` was used to filter to a peptide score of 0.05 with `-score:pep 0.05`

`IsobaricAnalyzer` with `-type itraq4plex` was used with the merged `.mzML` files to assign protein-peptide identifications to features or consensus features with `IDMApper`.
The files for each run output by `IDMapper` were then merged with `FileMerger`.
Bayesian score estimation and protein inference was performed with `Epifany` and the following flags: `-greedy_group_resolution remove_proteins_wo_evidence -algorithm:keep_best_PSM_only false` Decoys were removed and 0.05 FDR filtering was done via `IDFilter` with `-score:protgroup 0.05 -remove_decoys`.
Finally, `IDConflictResolver` was used to resolve ambiguous annotations of features with peptide identifications, before quantification with `ProteinQuantifier`.

## Label free OpenMS analysis {#openms-label-free}

For quantification, the raw spectra files were analysed via OpenMS (version 2.6.0) command line tools, with the workflow from the prior section (\@ref(openms-chap3)) adapted to suit a label-free analysis.
The files were first converted from the proprietary .Raw format to the open .mzML standard with the `FileConverter` tool via the open-source `ThermoRawFileParser`.[@rost_openms_2016; @hulstaert_thermorawfileparser_2020] Unless otherwise stated, default arguments were used throughout.

The decoy database generated in the prior section (iTRAQ OpenMS analysis) was also re-used.
The `CometAdapter` was used to run the search.[@eng_comet_2013] Fixed modifications were set to "Carbamidomethyl (C)" and "Oxidation (M)" was set as a variable modification.
To reflect the instrument the following flags were also set: `-precursor_mass_tolerance 20 -isotope_error 0/1`.

To annotate the identified peptides with proteins the `PeptideIndexer` tool was used.
`PeptideIndexer` and `PSMFeatureExtractor` were used for annotation.
For peptide level score estimation and filtering `PercolatorAdapter` was used with the following flags: `-score_type q-value -enzyme trypsin`.
`IDFilter` was used to filter to a peptide score of 0.01 with `-score:pep 0.01` followed by `IDScoreSwitcher` with the following flags: `-new_score "MS:1001493" -new_score_orientation lower_better -new_score_type "pep" -old_score "q-value"`.
The `ProteomicsLFQ` was used for subsequent processing with the flags: `-proteinFDR 0.05 -targeted_only true`.
The `-out_msstats` flag was also used to produce quantitative data for downstream statistical analysis with the R package `MSstats`.[@choi_msstats_2014]

## Network and pathway analysis {#pathway-analysis-chap3}

The Bioconductor package ReactomePA, which employs the open-source, open access, manually curated and peer-reviewed pathway database Reactome was used for network analysis.[@yu_reactomepa_2016; @jassal_reactome_2020]

## Enzyme-linked immunosorbent assays

Four proteins identified by the iTRAQ analysis were measured by enzyme-linked immunoabsorbent assay (ELISA) from non-pooled samples to validate the iTRAQ findings.

These proteins were alpha-2-macroglobulin (A2M), retinol binding protein 4 (RBP4), serum amyloid A1 (SAA1) and apolipoprotein A1 (ApoA1).
They were selected for their biological relevance and differential abundance between AIS C improvers and non-improvers, implying potential as biomarkers of neurological outcome prediction.
A2M, RBP4 and SAA1 were assessed using a human DuoSet® ELISAs (R&D Systems, Abingdon, UK).
ApoA1 was assessed using a human Quantikine® ELISA (R&D Systems, Abingdon, UK).
Samples were diluted 1:600,000 for A2M and RBP4, 1:100 for SAA1 and 1:20,000 for ApoA1 in the respective assay kit diluent.
Samples that were above the assay detection limit were rerun at 1:300 and 1:40,000 for SAA1 and ApoA1 respectively.
All ELISAs were carried out according to the manufacturer's protocol.
Protein concentrations were normalised to the sample dilution factor.
Statistical analysis was performed using the statistical programming language `r R.version$version.string`.
Pairwise t tests with bonferroni adjusted P-values with the R `rstatix` package were used to assess differential abundance.

# Results

Plasma from American Spinal Injury Association (ASIA) grade C SCI patients (total n=17) contrasting those who experienced an AISA grade conversion (n=10), and those who did not (n=7) collected within 2 weeks, and at approximately 3 months post-injury (Improvers n=9 vs Non-improvers n=6).
Relative protein abundance in AIS grade A (n=10) and grade D (n=11) patients was also examined.

In the interest of brevity, only the plots of acute and subactue AIS C improvers VS non-improvers are included here, please see the supplemental data for the other comparisons (section \@ref(sup-data)).

<!-- Example appendix references: see section @ref(sup-string-plots) -->

```{r}
## read in OpenMS data
heatmap <- readRDS("data/openms/reactomepa_heatmaps_quantification_output.rds")
cnetplot_itraq <- readRDS("data/openms/reactomepa_cnetplots_quantification_output_2021-01-18.rds")
volcano <- readRDS("data/openms/msstats_volcano_plots_boot.rds")
foldchange_list <- readRDS("data/openms/protein_quantification_foldchanges.rds")
## stringdb data
stringdb_results <- readRDS("data/openms/protein_quantification_stringdb_results_2021-03-23.rds")
## There are duplicate rows - remove them
## map(foldchange_list, nrow)
foldchange_list <- map(foldchange_list, ~ dplyr::distinct(.x))
foldchange_nrow <- map(foldchange_list, nrow)
```

<!-- ### Comparing OpenMS and ProteinPilot -->

```{r, label='openms-fc-table-prep'}
## merge list
fc_openms <- do.call(rbind, foldchange_list)
## add run column
fc_openms <- fc_openms %>%
    rownames_to_column() %>%
    dplyr::mutate(run = case_when(
                      grepl("run1", rowname) ~ "1",
                      TRUE ~ "2")) %>%
    dplyr::mutate(piv_wide = paste(comparison, run, sep = "_run"))

fc_openms$piv_wide <- as.factor(fc_openms$piv_wide)
levels(fc_openms$piv_wide) <- c("AIS A vs D", "AIS C improvers vs A",
                                "Acute AIS C improvers vs non-improvers",
                                "AIS C improvers vs non-improvers",
                                "AIS C improvers acute vs subacute",
                                "AIS C improvers vs D", "AIS C non-improvers vs A",
                                "AIS C non-improvers acute vs subacute",
                                "AIS C non-improvers vs D", "CROSS-RUN",
                                "Subacute AIS C improvers vs non-improvers",
                                "CROSS-RUN2", "CROSS-RUN3", "CROSS-RUN4")
## filter out runs
openms_run1 <- fc_openms %>%
    dplyr::filter(run == 1)
openms_run2 <- fc_openms %>%
    dplyr::filter(run == 2) %>%
    dplyr::filter(!grepl("subacute", comparison))
protein_up <- fc_openms %>%
    dplyr::filter(log2_fold_change > 0) %>%
    dplyr::group_by(piv_wide) %>%
    summarise(n = n())
protein_down <- fc_openms %>%
    dplyr::filter(log2_fold_change < 0) %>%
    dplyr::group_by(piv_wide) %>%
    summarise(n = n())
## remove uneeded cols and pivot wide
fc_openms_wide <- fc_openms %>%
    dplyr::filter(!grepl("CROSS-RUN", piv_wide)) %>%
    dplyr::select(!c(rowname, comparison, run, accession, fold_change, protein_score)) %>%
    pivot_wider(names_from = piv_wide, values_from = log2_fold_change) %>%
    dplyr::arrange(gene)
```

```{r, label='proteinpilot-fc-table-prep'}
## load proteinpilot fcs
load(file = "data/protein_descriptions_wide.rda")
## remove desctiption column
pp_fc <- protein_descriptions_wide %>%
    dplyr::select(!c(description, protein_count)) %>%
    ## select comparisons of interest
    dplyr::select_if( !grepl("c_improv_acute_vs_c_nonimprov_subacute|c_improv_subacute_vs_c_nonimprov_acute", names(.)))
## shortern the protein names
pp_fc$protein_name <- gsub( " .*$", "", pp_fc$protein_name, useBytes = TRUE)
## sort protein name col alphabetically
pp_fc <- pp_fc[order(pp_fc$protein_name), ]
## filter out rows where all remaining comparisons are NA
pp_fc_filtered <- pp_fc %>% filter_at(vars(c_improv_acute_vs_c_nonimprov_acute:a_vs_d),
                                      any_vars(!is.na(.)))
compar_cols <- c("Protein", "AIS C improvers acute vs subacute",
                 "Acute AIS C improvers vs non-improvers",
                 "Subacute AIS C improvers vs non-improvers",
                 "AIS C non-improvers acute vs subacute",
                 "AIS C improvers vs non-improvers",
                 "AIS C improvers vs A",
                 "AIS C improvers vs D",
                 "AIS C non-improvers vs A",
                 "AIS C non-improvers vs D",
                 "AIS A vs D")
names(pp_fc_filtered) <- compar_cols
```

```{r, label='openms-protp-venn-prep'}
## proteinpilot data pivot long
pp_long <- pp_fc_filtered %>%
    pivot_longer(!Protein, names_to = "comparison",
                 values_to = "fc", values_drop_na = TRUE) %>%
    dplyr::mutate(upregulated = case_when(fc > 0 ~ "up",
                                          fc < 0 ~ "down",
                                          TRUE ~ "ERROR"),
                  piv_wide = as.factor(comparison))
pp_updown <- pp_long %>%
    dplyr::group_by(piv_wide, upregulated) %>%
    dplyr::summarise(n = n ()) %>%
    dplyr::mutate(group = "ProteinPilot")
## openms
openms_long <- fc_openms %>%
    dplyr::select(gene, log2_fold_change, piv_wide) %>%
    dplyr::filter(!is.na(log2_fold_change)) %>%
    dplyr::mutate(upregulated = case_when(log2_fold_change > 0 ~ "up",
                                          log2_fold_change < 0 ~ "down",
                                          TRUE ~ "ERROR"))
openms_updown <- openms_long %>%
    dplyr::group_by(piv_wide, upregulated) %>%
    dplyr::summarise(n = n ()) %>%
    dplyr::mutate(group = "OpenMS")
## function for getting comparison lists
venn_list <- function(df1, df2, col, group) {
    ## subset dfs
    df1 <- df1[df1[col] == group, ]
    df2 <- df2[df2[col] == group, ]
    ## make venn list
    venn_list <- list(OpenMS = df1$gene,
                      ProteinPilot = df2$Protein)
    return(venn_list)
}
updown_compare <- map(unique(pp_long$comparison),
                      ~ venn_list(openms_long, pp_long, "piv_wide", .x))
names(updown_compare) <- unique(pp_long$comparison)
```

<!-- not included in karinas edit -->

<!-- AIS C improvers had `r as.numeric(protein_up[(protein_up$piv_wide == "Acute AIS C improvers vs non-improvers"), 2])` more abundant proteins, and `r as.numeric(protein_down[(protein_down$piv_wide == "Acute AIS C improvers vs non-improvers"), 2])` less abundant proteins at the acute phase with OpenMS, as opposed to `r as.numeric(pp_updown[(pp_updown$piv_wide == "Acute AIS C improvers vs non-improvers"),3][2,])` and `r as.numeric(pp_updown[(pp_updown$piv_wide == "Acute AIS C improvers vs non-improvers"),3][1,])` with ProteinPilot (Figure @ref(fig:openms-pp-updown)). -->

<!-- At the subacute phase, AIS C improvers had `r as.numeric(protein_up[(protein_up$piv_wide == "Subacute AIS C improvers vs non-improvers"), 2])` and `r as.numeric(pp_updown[(pp_updown$piv_wide == "Subacute AIS C improvers vs non-improvers"),3][2,])` proteins of increased abundance were found, whereas `r as.numeric(protein_down[(protein_down$piv_wide == "Subacute AIS C improvers vs non-improvers"), 2])` and `r as.numeric(pp_updown[(pp_updown$piv_wide == "Subacute AIS C improvers vs non-improvers"),3][1,])` proteins were less abundant, for OpenMS and ProteinPilot respectively (Figure @ref(fig:openms-pp-updown)). -->

<!-- MH note: I've commented out the above, the summarised version was didn't have enough detail so I've removed it -->

(ref:openms-pp-venn) Venn diagram of the overlap in unique proteins identified from analysis with ProteinPilot and OpenMS respectively on data from 2 4-plex iTRAQ experiments.

```{r, label='openms-pp-venn', fig.cap='(ref:openms-pp-venn)', fig.align = "center", eval = FALSE, out.width="60%"}
## make list of proteins from each
x <- list(OpenMS = fc_openms_wide$gene, ProteinPilot = pp_fc_filtered$Protein)
## plot
ggVennDiagram(x, label_alpha = 0) +
    ## scale_fill_gradient(low="blue",high = "red") +
    theme(legend.position = "none")
```

(ref:openms-pp-updown) The number of proteins found to be up- or down-regulated via ProteinPilot and OpenMS.

```{r, label='openms-pp-updown', fig.cap='(ref:openms-pp-updown)', eval = FALSE}
## rbind data
updown <- rbind(openms_updown, pp_updown) %>%
    dplyr::filter(!grepl("CROSS", piv_wide))
## plot
updown %>%
    ggplot(aes(x = upregulated, y = n, fill = group)) +
    geom_bar(stat = "identity", position = position_dodge()) +
    facet_wrap( ~ piv_wide, labeller = labeller(piv_wide = label_wrap_gen(25))) +
    labs(x = expression(log[2] ~ fold - change),
         y = "Number of Proteins") +
    custom_theme() +
    theme(axis.text.x = element_text(angle = 0, vjust = 0, hjust = 0.5),
          legend.position = 'top', legend.direction = "horizontal") +
    scale_fill_manual(values = c("tomato", "steelblue"), name = "")
```

## iTRAQ analyses

```{r, label='load-fc-data-chap4'}
## get foldchange data
load("data/label_free_openms_foldchange_df.rda")
## filter to significant proteins
lf_sig <- foldchange_df %>%
  dplyr::filter(pvalue <= 0.05) %>%
  dplyr::select(Label, log2FC, gene_name)

## there's a duplicate row, remove it
lf_sig <- lf_sig %>% dplyr::filter(!(gene_name == "ITIH4" &
  Label == "Acute C Improvers Vs Subacute D" &
  log2FC < 2))

names(lf_sig)[3] <- "Protein"
## need to split up table for space
lf_labels <- unique(lf_sig$Label)
lf_sig1 <- lf_sig %>%
  dplyr::filter(Label %in% lf_labels[1:(length(lf_labels)/2)])

lf_sig2 <- lf_sig %>%
  dplyr::filter(Label %in% lf_labels[(length(lf_labels)/2+1):length(lf_labels)])

lf_sig_wide1 <- lf_sig1 %>%
  pivot_wider(names_from = Label, values_from = log2FC) %>%
  dplyr::arrange(Protein)

lf_sig_wide2 <- lf_sig2 %>%
  pivot_wider(names_from = Label, values_from = log2FC) %>%
  dplyr::arrange(Protein)
```

## Differential protein abundances

AIS C improvers had 18 more abundant proteins and 49 less abundant proteins at the acute phase relative to non-improvers.
Similarly, at the subacute phase, AIS C improvers had 34 more abundant proteins and 34 less abundant proteins relative to non-improvers.
The AIS A group had 56 more abundant and 9 less abundant proteins respectively relative to non-improvers.
Acutely, AIS C improvers relative to AIS A and D had 21 and 53 more abundant and 46 and 12 less abundant proteins.

Please see the Tables \@ref(tab:openms-fc-table), \@ref(tab:label-free-fc-table1) & \@ref(tab:label-free-fc-table2) for a full list of protein fold change changes.

## Heatmaps {#heatmaps-chap3}

The majority of the pathways associated with the proteins identified by these iTRAQ experiments are related to the complement cascade and platelet activity (Figure \@ref(fig:openms-hmap-acute-c-1), \@ref(fig:openms-hmap-subacute-c-1)).
There are also several pathways implicated in metabolic processes, particularly with apolipoproteins and retinoids.

```{=tex}
\clearpage
\blandscape
```
(ref:openms-hmap-acute-c-1) Heatmap denoting the log$_2$ fold change of proteins in plasma collected 2-weeks post-injury, and the biological pathways these proteins are associated with on Reactome. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not.

```{r, label='openms-hmap-acute-c-1', fig.cap='(ref:openms-hmap-acute-c-1)', fig.align="center", fig.asp = 0.5, fig.width = 11}
heatmap$acute_c_improvers_vs_nonimprovers_run1 +
  theme(axis.text = element_text(size=6))
```

```{=tex}
\elandscape
\blandscape
```
(ref:openms-hmap-subacute-c-1) Heatmap denoting the log$_2$ fold change of proteins in plasma collected 3-months post-injury, and the biological pathways these proteins are associated with on Reactome. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not.

```{r, label='openms-hmap-subacute-c-1', fig.cap='(ref:openms-hmap-subacute-c-1)', fig.align="center", fig.asp = 0.5, fig.width = 11}
heatmap$subacute_c_improvers_vs_nonimprovers_run1 +
  theme(axis.text = element_text(size=6))
heatmap_itraq <- heatmap
```

```{r, label='openms-hmap-subacute-c-12', fig.cap='(ref:openms-hmap-subacute-c-1)', fig.align="center", fig.width = 14, fig.height = 7, eval=FALSE}
heatmap$subacute_c_improvers_vs_nonimprovers_run1
heatmap_itraq <- heatmap
```

```{=tex}
\elandscape
\clearpage
```
Similarly to the iTRAQ data, many of the Reactome pathways are associated with the complement cascade and platelets activation (Figures \@ref(fig:openms-chap4-hmap-acute-c-1), \@ref(fig:openms-chap4-hmap-subacute-c-1)).

\blandscape

(ref:openms-chap4-hmap-acute-c-1) Heatmap denoting the log$_2$ fold change of proteins in plasma collected 2-weeks post-injury, and the biological pathways these proteins are associated with on Reactome. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not. Grey blocks denote proteins not present in the comparison.

```{r, label='openms-chap4-hmap-acute-c-1', fig.cap='(ref:openms-chap4-hmap-acute-c-1)', fig.align="center", fig.asp = 0.5, fig.width = 10.5}
## read in heatmaps
load("data/openms_reactomepa_heatmaps_output.rda")
heatmap$acute_c_imp_vs_acute_c_nonimp +
  theme(axis.text = element_text(size=7))
```

```{=tex}
\elandscape
\blandscape
```
(ref:openms-chap4-hmap-subacute-c-1) Heatmap denoting the log$_2$ fold change of proteins in plasma collected 3-months post-injury, and the biological pathways these proteins are associated with on Reactome. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not. Grey blocks denote proteins not present in the comparison.

```{r, label='openms-chap4-hmap-subacute-c-1', fig.cap='(ref:openms-chap4-hmap-subacute-c-1)', fig.align="center", fig.asp = 0.5, fig.width = 10.5}
heatmap$subacute_c_imp_vs_subacute_c_nonimp +
  theme(axis.text = element_text(size=7))
```

```{=tex}
\elandscape
\clearpage
```
## Network analysis of Differentially Abundant Proteins between AIS C improvers and non-improvers {#cnetplot-chap3}

Similar to the heatmaps, network plots highlighted that the majority of proteins changes were associated with the complement cascade and pathways linked to platelet activity (Figure \@ref(fig:openms-cnetp-acute-c-1), \@ref(fig:openms-cnetp-subacute-c-1)).
Several proteins were also associated with the regulation of insulin-like growth factor.

\blandscape

(ref:openms-cnetp-acute-c-1) Network plot denoting the log$_2$ fold change of proteins in plasma collected 2-weeks post-injury, and the biological pathways these proteins are associated with on Reactome. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not.

```{r, label='openms-cnetp-acute-c-1', fig.cap='(ref:openms-cnetp-acute-c-1)', fig.align="center", fig.asp = 0.5, fig.width = 11}
cnetplot_itraq$acute_c_improvers_vs_nonimprovers_run1
```

```{=tex}
\elandscape
\blandscape
```
(ref:openms-cnetp-subacute-c-1) Network plot denoting the log$_2$ fold change of proteins in plasma collected 3-months post-injury, and the biological pathways these proteins are associated with on Reactome. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not.

```{r, label='openms-cnetp-subacute-c-1', fig.cap='(ref:openms-cnetp-subacute-c-1)', fig.align="center", fig.asp = 0.5, fig.width = 11}
cnetplot_itraq$subacute_c_improvers_vs_nonimprovers_run1
```

\elandscape

\clearpage

Similarly to the heatmaps and the iTRAQ data, network plots derived using the label-free data highlight the majority of differential proteins are associated with the complement cascade and pathways linked to platelets (Figures \@ref(fig:openms-chap4-cnetp-acute-c-1), \@ref(fig:openms-chap4-cnetp-subacute-c-1)).

\blandscape

(ref:openms-chap4-cnetp-acute-c-1) Network plot denoting the log$_2$ fold change of proteins in plasma collected 2-weeks post-injury, and the biological pathways these proteins are associated with on Reactome. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not.

```{r, label='openms-chap4-cnetp-acute-c-1', fig.cap='(ref:openms-chap4-cnetp-acute-c-1)', fig.align="center", fig.asp = 0.5, fig.width = 11}
## load data
load("data/openms_reactomepa_cnetplots_output_2021-08-09.rda")
cnetplot$acute_c_imp_vs_acute_c_nonimp
```

```{=tex}
\elandscape
\blandscape
```
(ref:openms-chap4-cnetp-subacute-c-1) Network plot denoting the log$_2$ fold change of proteins in plasma collected 3-months post-injury, and the biological pathways these proteins are associated with on Reactome. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not.

```{r, label='openms-chap4-cnetp-subacute-c-1', fig.cap='(ref:openms-chap4-cnetp-subacute-c-1)', fig.align="center", fig.asp = 0.5, fig.width = 11}
cnetplot$subacute_c_imp_vs_subacute_c_nonimp
```

```{=tex}
\elandscape
\clearpage
```
## Pathway analysis of Differentially Abundant Proteins between AIS C improvers and non-improvers {#kegg-chap3}

Pathway analysis via the `pathview` R package returned the complement and coagulation cascade to be on the sole significant KEGG pathway to derive from the OpenMS analysed data.
The majority of the proteins present in this pathway were less abundant in the 2-week post-injury plasma of AIS C patients who experienced an AIS grade conversion and those who did not (Figure \@ref(fig:kegg-complement)).

(ref:kegg-complement) KEGG complement cascade pathway annotated with log$_2$ fold change of proteins in plasma collected 2-weeks post-injury. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not.

```{r, label='kegg-complement', fig.cap='(ref:kegg-complement)', fig.align="center"}
knitr::include_graphics("figures/kegg_pathways/hsa04610_pathview.png")
```

Similarly to the iTRAQ pathway analysis, the label free data analysed via the pathview R package returned the complement and coagulation cascade to be the sole significant KEGG pathway derived from the OpenMS analysed data.
The majority of the proteins present in this pathway were less abundant 2-weeks post-injury in the plasma of AIS C patients who experienced an AIS grade conversion than those who did not (Figure \@ref(fig:kegg-complement-chap4)).

(ref:kegg-complement-chap4) KEGG complement cascade pathway annotated with log$_2$ fold change of proteins in plasma collected 2-weeks post-injury. This compares AIS C SCI patients who experienced an AIS grade improvement and those who did not.

```{r, label='kegg-complement-chap4', fig.cap='(ref:kegg-complement-chap4)', fig.align="center"}
knitr::include_graphics("figures/kegg_pathways/hsa04610.pathview_label-free.png")
```

## Validation of Proteomic Data using ELISA

No statistically significant difference between groups for A2M abundance in plasma via DuoSet® ELISAs, though there were outliers in the AIS A and D groups, and particularly in the AIS C patients at 3-months who did not experience an AIS grade conversion (Figure \@ref(fig:patchwork-2)).

A significant difference was found between AIS C non-improvers at 2-weeks and AIS D for SAA1, with outliers in AIS C non-improvers at 2-weeks, and both AIS C improvers and non-improvers at 3-months post-injury (Figure \@ref(fig:patchwork-2)).
For ApoA1 plasma abundance estimated via Quantikine® ELISAs, statistically significant differences were found between AIS C improvers at 2-weeks and both AIS C improvers and non-improvers at 3-months, AIS C 3-month improvers and AIS A and D, and AIS C 3-month non-improvers and AIS A and D (Figure \@ref(fig:patchwork-2)).
A statistically significant difference was also found between AIS C improvers and non-improvers at 2-weeks post-injury for RBP4 (Figure \@ref(fig:patchwork-2)).

(ref:patchwork-2) Normalised estimated concentration of $\alpha$-2-macroglobulin (A), serum amyloid A1 (B), apolipoprotein A1 (C) and retinol binding protein 4 (D). Estimates were calculated from the optical density of a standard curve produced via a DuoSet® ELISA. Plasma from each patient that made up the pooled iTRAQ samples was assayed and pairwise t-tests with bonferroni adjusted P-values were performed to assess differential abundance.

```{r, label='elisa-sig-plots1'}
## get the plot objects to be put into a grid
elisa_sig_plots <- readRDS("data/elisas/sig_plots_2021-03-16.rds")
```

```{r, label='patchwork-2', fig.cap='(ref:patchwork-2)', fig.height = 10}
patchwork_plots <- Reduce("/",elisa_sig_plots)

## Remove x axis from first and second subplots
patchwork_plots[[1]] = patchwork_plots[[1]] + theme(axis.text.x = element_blank(),
                                                    axis.title.x = element_blank()) +
  ylab(str_wrap(str_replace(patchwork_plots[[1]]$labels$y, "foo", " "), width = 25))
patchwork_plots[[2]] = patchwork_plots[[2]] + theme(axis.text.x = element_blank(),
                                                    axis.title.x = element_blank()) +
  ylab(str_wrap(str_replace(patchwork_plots[[2]]$labels$y, "foo", " "), width = 25))
## remove title from third and fourth x axis
patchwork_plots[[3]] = patchwork_plots[[3]] + theme(axis.text.x = element_blank(),
                                                    axis.title.x = element_blank()) +
  ylab(str_wrap(str_replace(patchwork_plots[[3]]$labels$y, "foo", " "), width = 25))
patchwork_plots[[4]] = patchwork_plots[[4]] + theme(axis.title.x = element_blank()) +
  ylab(str_wrap(str_replace(patchwork_plots[[4]]$labels$y, "foo", " "), width = 25))
## add letter tags
patchwork_plots +
  plot_annotation(tag_levels = 'A')
```

```{r, label='fc-data-chap4'}
## get all unique proteins for venn diagram
label_free_proteins <- unique(foldchange_df$gene_name)
groups <- unique(foldchange_df$abbrev_labels)
## filter to foldchange greater than 1.2
foldchange_df <- foldchange_df %>% dplyr::filter(log2FC > 1.2 | log2FC < -1.2)
## summary df with folchange counts
fc_summary_df <- foldchange_df %>%
    dplyr::mutate(up_reg = if_else(log2FC > 0, 1, 0),
                  down_reg = if_else(log2FC < 0, 1, 0)) %>%
    dplyr::group_by(abbrev_labels) %>%
    dplyr::summarise(n = n(), up_reg = sum(up_reg),
                     down_reg = sum(down_reg))

pos_neg_filter <- function(comparison, df) {
    ## filter to comparison and negative foldchange
    df_neg <- df %>%
        dplyr::filter(abbrev_labels == comparison & log2FC < 0)
    ## filter to comparison and positive foldchagne
    df_pos <- df %>%
        dplyr::filter(abbrev_labels == comparison & log2FC > 0)

    df_list <- list("pos" = df_pos, "neg" = df_neg)
    return(df_list)
}
pos_neg <- map(groups, ~ pos_neg_filter(.x, foldchange_df))
names(pos_neg) <- groups
```

```{=html}
<!-- The data processed by `MSstats` was filtered to proteins with a adjusted P-value of < 0.05 and a log$_2$ FC of > $\pm1.2$.
The total number of significant proteins is `nrow(foldchange_df)`. -->
```

## Comparing iTRAQ and label-free proteins

A total of `r length(label_free_proteins)` and `r nrow(fc_openms_wide)` unique proteins were identified across the label-free and iTRAQ experiments respectively, with a modest overlap of 26 proteins found using both techniques.

<!-- MH note: ask KW is bar plot would be better than venn diagram? -->

(ref:itraq-label-free-venn) Venn diagram of the overlap in unique proteins identified from iTRAQ and label-free proteomic experiments analysed via OpenMS.

```{r, label='itraq-label-free-venn', fig.cap='(ref:itraq-label-free-venn)', out.width="60%", fig.align="center", eval = FALSE}
## make list of proteins from each
x <- list("iTRAQ" = fc_openms_wide$gene, "Label-free" = label_free_proteins)
save(x, file = "data/itraq_vs_label-free_venn_list.rda")
## plot
ggVennDiagram(x, label_alpha = 0) +
    ## scale_fill_gradient(low="blue",high = "red") +
    theme(legend.position = "none")
```

```{r, label='up-down-itraq-label-free'}
## get the comparistors to be the same
itraq_up_down <- fc_openms
groups_itraq <- unique(itraq_up_down$comparison)
itraq_up_down$comparison <- itraq_up_down$comparison %>%
    str_replace("c_acute_improve", "acute_c_imp") %>%
    str_replace("c_acute_nonimprove", "acute_c_nonimp") %>%
    str_replace("c_subacute_improve", "subacute_c_imp") %>%
    str_replace("c_subacute_nonimprove", "subacute_c_nonimp") %>%
    str_replace("vs_a", "vs_acute_a") %>%
    str_replace("a_vs", "acute_a_vs") %>%
    str_replace("vs_d", "vs_acute_d") %>%
    str_replace("d_vs", "acute_d_vs")
## some of the comparisons are the other way around - would need to fix
```

# Discussion

This is the first study, to our knowledge, to investigate the plasma proteome in SCI patients whose AIS scores either improved or did not improve post injury and also to compare these to AIS grade A and D patients.
We have used two proteomic techniques allowing us to profile both high and low abundance proteins, in order to identify protein candidate biomarkers which may have potential to predict neurological improvement within the acute setting.
Moreover, this data can better inform us of the biology underlying neurological improvement or stability in a cohort of patients being conservatively managed post SCI.

Briefly, for processing of proteomic data, we compared the performance of the mass spectrometry vendor (ABSciex) provided ProteinPilot (version 4.5) and OpenMS (version 2.6.0).
As there were only modest difference in both the proteins identified and the respective fold changes (data not shown), we opted to use OpenMS for the greater transparency and reproducibility it offers as open-source software.

This study has highlighted a number of proteins that may be able to discriminate in, the acute phase following injury, between AIS grade C patients who either improve or do not improve by an AIS grade following SCI.
The most promising of these is Retinol Binding Protein 4 (RBP4) which was demonstrated to be increased in non-improvers compared to improvers in the acute phase.
Further this change could be confirmed using ELISA, which may provide a more clinically useful means of assessment on a wide scale.

RBP4 is synthesised in the liver and binds retinol that is released following vitamin A deficiency.[@peterson_studies_1971] 
Once delivered to target cells, retinol can either be converted to retinaldehyde, which is required for functional vision, or oxidised to retinoic acid, which is a ligand for nuclear receptors, thus regulating gene expression.[@lane_role_2005; @balmer_gene_2002] 
The role of retinoid signalling in spinal cord and motor neuron differentiation, including development of regions of the spinal cord has been outlined, and implies a possible involvement in maintaining motor neuron integrity.[@colbert_retinoid_1995; @sockanathan_motor_1998] 
The mRNA of a rodent homologue of RBP was found to be up-regulated at 24 hours post-SCI and may promote cell proliferation and regeneration by increasing retinoid metabolism.[@song_genechip_2001; @hurst_complexity_1999]

Another study of amyotrophic lateral sclerosis (ALS), a neurodegenerative disease, comparing gene expression between post-mortem spinal cord samples of ALS and controls similarly observed up-regulation of RBP1 in ALS spinal cord.[@malaspina_differential_2001] 
Furthermore, a transgenic mouse study reported retinoid signalling may contribute to the retained plasticity and regenerative potential of the mature spinal cord.[@haskell_retinoic_2002] 
Collectively, these results might support a hypothesis that AIS C improvers have increased levels of RBP4 and this relates to improved capacity for neuronal regeneration/plasticity.
Whether this is due to increased expression or due to higher vitamin A dietary intake is unclear from this data.

<!-- MH NOTE: should probably temper the above line by mentioning that the ELISA wasn't significant... -->

Alongside RBP4, a number of other protein abundance differences across the different biological comparisons were identified in proteins associated with liver function.
Our previous work investigating the potential of routinely measured haematological analytes for predicting neurological outcome in SCI patients also highlighted several proteins that were linked with liver function; thus providing further support to the theory that liver status is relevant to differential functional recover.[@brown_preliminary_2019; @bernardo_harrington_routinely_2020] 
The pathway analysis specifically indicated that the acute phase response (APR) is implicated.

The APR is the bodies first response to infection or injury, including SCI.
This systemic response is largely coordinated by factors released from the liver, but the APRs effects extend to multiple peripheral organs including the kidneys, lungs and spleen.[@bao_systemic_2012; @campbell_liver_2008; @fleming_remote_2012; @gris_systemic_2008] 
This hepatic response is typically transient and quickly fades, but prolonged liver inflammation and pathology has been observed in rodent SCI models.[@goodus_dietary_2018; @sauerbeck_spinal_2014] 
Basic liver functions are chronically impaired by SCI, including metabolising carbohydrates, fats and proteins, storage of minerals vitamins and glycogen and filtering blood from the digestive tract.[@garcia-lopez_acute_2007; @deleve_hepatic_2007; @farkas_neurogenic_2018; @chow_pharmacology_2012; @sauerbeck_spinal_2014]

The acute (1-7 days) liver response to SCI is well documented; the inflammatory cytokines including TNF$\alpha$, IL-1$\alpha$, IL-1$\beta$ and IL-6, released at the injury site, reach the liver through the bloodstream.[@fleming_remote_2012; @hundt_assessment_2011] 
This provokes the liver to enter the APR and produce acute phase proteins thus stimulating a greater immune response.[@anthony_systemic_2014; @fleming_remote_2012] 
The hepatocytes that make up the majority of the liver biomass, express receptors that bind the aforementioned inflammatory cytokines; similarly the hepatic macrophage Kupffer cells also bind these cytokines, complement proteins and lipopolysaccharide (LPS) and swiftly remove microorganisms, endotoxins and other debris from the blood.[@yang_clec4f_2013; @szalai_complement-dependent_2000; @crispe_hepatocytes_2016; @campbell_central_2005] 
Furthermore, it has been suggested that liver inflammation and Kupffer cells activity promote recruitment of leukocytes to the injury site in brain or spinal trauma, potentially enhancing CNS injury.[@anthony_systemic_2014; @campbell_central_2005] 
For example, a rodent study demonstrated depletion of Kupffer cells prior to injury resulted in few neutrophils infiltrating the injury site.[@campbell_liver_2008; @campbell_hepatic_2008]

Another protein that out label-free proteomic data highlights is Peroxiredoxin 2 (PRX-2), which was detected acutely in AIS C improvers and AIS D patients, and subacutely in AIS A and AIS D. Peroxiredoxins are a large and highly conserved family of enzymes that reduce peroxides.
PRX-2 is highly abundant in RBCs and intracellularly serves as an important anti-oxidant role in various cells types, including neurons.[@low_peroxiredoxin_2008] 
By contrast, extracellular PRX-2 has been suggested to act as an inflammatory DAMP, leading microglia and macrophages to release a plethora of pro-inflammatory factors.[@salzano_linkage_2014; @garcia-bonilla_peroxiredoxin_2012; @shichita_peroxiredoxin_2012] 
An *in vitro* primary neurons and microglia co-culture study reported PRX-2 activating microglia via TLR-4, potentially leading to neuronal apoptosis.[@lu_peroxiredoxin_2018] 
A murine study found over-expression of PRX-2 attenuated oxidative stress and neuronal apoptosis following subarachnoid haemorrhage.[@lu_peroxiredoxin_2019] 
Over-expression of PRX-2 is speculated to protect again ischaemic neuronal injury by modulating the redox-sensitive thioredoxin-apoptosis signal-regulating kinase (ASK) 1 signalling complex.[@gan_transgenic_2012] 
Several molecular chaperones can interact with ASK1, including thioredoxin and TNF receptor-associated factor 6.[@matsuzawa_ros-dependent_2005] 
The dissociation of the thioredoxin-ASK1 complex activates ASK1.
PRX-2 is oxidised after scavenging free radicals, whereupon its antioxidantive activity is reduced.
This inactivation can be reversed by the thioredoxin-thioredoxin reductase system, whereby oxidised PRX-2 can regain its activity by reducing thioredoxin, leading to the dissociation of the thioredoxin-ASK1 complex.[@rhee_multiple_2011] 
Additionally, oxidised PRX-1 can inhibit ASK1-induced apoptosis via the thioredoxin-binding domain on ASK1.[@kim_novel_2008]

The presence of PRX-2 in acute AIS C improvers and absence in acute C non-improvers suggests the protein could indicate a more protective action against oxidative stress, and implies the protein has potential value as a biomarker of functional outcomes.
Similarly, PRX-2 may be acting as a healthy response to trauma-induced oxidative stress in both acute AIS D, although the persistence to the subacute time-point is less clear.
Likewise, the presence of PRX-2 in AIS A subacutely, but not acutely is more perplexing.
It should be noted that as plasma was used and cells were lysed, there is no distinction between intracellular and extracellular PRX-2 in this data.
Perhaps in the more severe AIS A injury, secondary injuries, including oxidative stress, are greater and so persist to the subacute time-point.
The acute absence may be a result of an overwhelmed physiology unable to respond or prioritise managing oxidative stress.

Pathway analysis from both the iTRAQ and label-free experiments identified the complement and coagulation cascades as a significant pathway of interest.
More broadly, the trend in this data is for proteins in the complement pathway is lower abundance, or inhibitory proteins such as C4BP to be more abundant, in the acute improvers.
C3 for instance, cleavage of which is vital for complement activation, was less abundant in acute AIS C improvers relative to non-improvers.
This finding is in line with a genetic C3 knockout study in mice which reported better neurological scores 2 days post-injury, reduced residual consolidated neurological deficit at 21 days and display minor change in reduced gliosis (20% decrease at 1h timepoint) but a three-to-fourfold decrease in neutrophil infiltration, resulting in enhanced regeneration of axons.[@qiao_complement_2006] 
Another study using a similar C3 knockout model reported improved neurological scores at acute and long-term time points.[@guo_effects_2010] 
These results imply that the complement cascade is a particularly important component of a differential response to neurological injury which ultimately leads to greater functional recovery.
Given the complexity of the complement cascade and the limited time points in this study, further work is needed to elucidate which facets of the cascade are outcome modifying, and at which stages post-injury.

AIS A and D samples were included largely to compare to the AIS C improvers.
If the improvers were all just a less severe AIS C, we might expect them to be more similar to the AIS D samples, with the non-improvers being closer to the As.
As this is not what we observed, we can conclude there is more to the differential functional recovery than initial injury severity.

The small number of statistically significant proteins speaks to the variability of human plasma samples, and is likely exacerbated by the inconstant timing of sample collection relative to injury.
Thus, a repeat of this experiment with a larger sample size will likely reveal many more proteins of potential interest.
Regardless, this study has highlighted RPB4 and PRX-2 as potential biomarkers of functional recover.
We have also highlighted the complement cascade as being a particularly important pathway in differential recovery.
Additional investigation of these proteins, but also the complement cascade more broadly, particularly at more acute time points, would also be valuable.
Furthermore, a metabolomic analysis with a similar samples would greatly compliment this work, particularly with regards to investigating further links to the livers role in neurological recovery.
Similarly, this additional work could complement the AIS A and D sample data, which may reveal further insights from this data.

\newpage

# Supplementary material {#sup-data .unnumbered}

```{=tex}
\beginsupplement
```

## Session Information

```{r session-info, echo=FALSE}
R.version
```

\clearpage

```{r package-table, echo=FALSE}
# devtools::session_info()
package_versions = function(packages=NULL, base=TRUE, sort=FALSE) {
    # extract package versions
    if (is.null(packages))
        packages <- names(sessionInfo()$otherPkgs)
    if (base)
        packages <- c("base", packages)
    if (sort)
        packages <- sort(packages)
    if (!is.null(packages)) {
        tmp <- data.frame(sapply(sapply(packages, packageVersion, simplify=FALSE), as.character))
        tmp2 = data.frame(sapply(sapply(packages, packageDate, simplify=FALSE), as.character))
        colnames(tmp) = "version"
        colnames(tmp2) = "date"
        data.frame(package=rownames(tmp), version=tmp[1], date=tmp2[1], row.names=NULL)
    }
}
package_table = package_versions()
kbl(package_table, booktabs = T, caption = "Packages Used")
```

\clearpage

```{r, label='openms-fc-table'}
kbl(fc_openms_wide, "latex", longtable = T, booktabs = T, escape = F,
             caption = "OpenMS log$_2$ fold changes in the plasma proteome of SCI patients from iTRAQ experiments. 'Acute' and 'Subacute' samples collected within 2 week and approximately 3-months post-injury respectively.") %>%
    kable_styling(latex_options =c("repeat_header"), font_size = 5) %>%
    column_spec(2:ncol(fc_openms_wide), width = "1.6cm") %>%
    column_spec(1, width = "0.5cm") %>%
    ## kable_styling(latex_options =c("scale_down")) %>%
    landscape()
```

```{r, label='label-free-fc-table1'}
kbl(lf_sig_wide1, "latex",
  longtable = T, booktabs = T, escape = F,
  caption = "OpenMS log$_2$ fold changes in the plasma proteome of SCI patients from label-free experiments. 'Acute' and 'Subacute' samples collected within 2 week and approximately 3-months post-injury respectively."
) %>%
  kable_styling(latex_options = c("repeat_header"), font_size = 5) %>%
  column_spec(2:ncol(lf_sig_wide1), width = "1.6cm") %>%
  column_spec(1, width = "0.5cm") %>%
  landscape()
```

```{r, label='label-free-fc-table2'}
kbl(lf_sig_wide2, "latex",
  longtable = T, booktabs = T, escape = F,
  caption = "OpenMS log$_2$ fold changes in the plasma proteome of SCI patients from label-free experiments. 'Acute' and 'Subacute' samples collected within 2 week and approximately 3-months post-injury respectively."
) %>%
  kable_styling(latex_options = c("repeat_header"), font_size = 5) %>%
  column_spec(2:ncol(lf_sig_wide2), width = "1.6cm") %>%
  column_spec(1, width = "0.5cm") %>%
  landscape()
```

# References
